lisp_ease/src/queries/package.lisp

```diff
@@ -0,0 +1,3 @@
+(defpackage :queries-pkg
```

Comment:
When files from ``queries`` go up into ``src``, this file should be removed and corresponding ``use-package`` calls in other files should be updated.